apps/AppWithWearable/lib/pages/home/page.dart

```diff
@@ -344,6 +390,86 @@ class _HomePageWrapperState extends State<HomePageWrapper> with WidgetsBindingOb
                 ),
               ),
               // Text(['Memories', 'Device', 'Chat'][_selectedIndex]),
+              if (_controller!.index == 2)
```

Comment:
![IMG_3714](https://github.com/BasedHardware/Friend/assets/18078879/823a2571-d43e-4439-bbbb-17225d7e93ca)

This looks bad, I don't see how this fits with current designs besides de border gradient.

And let's please remove the padding GPT comments.

Put this in a separate widget.

Finally, please put this component in a separate function, could be inside same file.

Remove the "Default" and put only "View Plugins"

Also, messages answered with plugin personality should have the plugin id as optional field in db, and instead of the message sender icon we have on the left of ai message it should use the plugin image as the sender.

Reply:
Please explain how you want the widget to look like. If you don't set specific deliverables I don't know what you want. You say it looks ugly and doesn't fit in current app designs. Is it the font? Spacing? Size? Palette? I can iterate on it but *I need to know* what you want. Otherwise this is not going to work. Please advise